src/modules/todo.js

```diff
@@ -0,0 +1,103 @@
+const taskContainer = document.querySelector('.task-container');
+
+export default class myTodo {
+  constructor() {
+    this.tasks = [];
+  }
+
```

Comment:
- [ ] As mentioned by the previous reviewer, it is among the project requirements **to implement the functionality of editing the task description and also save the changes in the local storage so that the next time you reload the page, they will be reflected on the page**. That means if the task description is for example `go to the gym` then the user may edit the description to for example `complete the project` so please implement that functionality 🙏 
Here is the point in the project description 👇 
![image](https://user-images.githubusercontent.com/90222110/184377868-4ce4b3ad-d930-4959-b7d7-29d977193a7c.png)